src/System.Private.ServiceModel/tools/test/BridgeCertificateBootstrapper/Bootstrapper.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Infrastructure.Common;
+
+public static class BridgeCertificateBootstrapper
+{
+    public static int Main(string[] args)
+    {
+        DisplayBanner(); 
+
+        if (args.Length < 1)
+        {
+            DisplayUsage();
+            return -1;
+        }
+
+        var filename = args[0];
+
```

Comment:
Is it possible we might have to do Directory.Create of the directory where this file will be written?  I'm thinking usage might name an outputfile in a non-existent folder.  Alternatively consider testing for existence of the folder before attempting the write.